nixos/modules/services/x11/xserver.nix

```diff
@@ -13,7 +13,8 @@ let
 
   # Map video driver names to driver packages. FIXME: move into card-specific modules.
   knownVideoDrivers = {
-    virtualbox = { modules = [ kernelPackages.virtualboxGuestAdditions ]; driverName = "vboxvideo"; };
```

Comment:
@vcunat I think we should leave in virtualbox guest additions for other things. I believe it provides more than just the driver. If it's listed last, I think it will get lower precedence in the case of conflicts.

Reply:
Actually, didn't realize this only effects xserver, so disregard the previous comment.